dojo/engine_tools/views.py

```diff
@@ -380,7 +387,23 @@ def edit_finding_exclusion_request(request: HttpRequest, fxid: str) -> HttpRespo
     
     return redirect('edit_finding_exclusion', fxid=fxid)
         
-        
+
+def delete_finding_exclusion(request: HttpRequest, fxid: str) -> HttpResponse:
```

Comment:
according to the exclusion of findings update the findings that exist with the exclusion removed.